.gitlab-ci.yml

```diff
@@ -96,24 +97,19 @@ BuildArborXMaster:
             -DARBORX_ENABLE_BENCHMARKS=ON
             -DARBORX_PERFORMANCE_TESTING=ON .. &&
       make ArborX_BoundingVolumeHierarchy.exe
-    - cp ./benchmarks/bvh_driver/ArborX_BoundingVolumeHierarchy.exe /ccsopen/proj/csc333/ArborX_BoundingVolumeHierarchyMaster${MASTER_HASH}.exe
-    - echo export MASTER_HASH=${MASTER_HASH} > ${CI_PROJECT_DIR}/master_hash
+    - cp ./benchmarks/bvh_driver/ArborX_BoundingVolumeHierarchy.exe /ccsopen/proj/csc333/ArborX_BoundingVolumeHierarchyMaster${BRANCH_HASH}.exe
   tags:
     - nobatch
-  artifacts:
-    paths:
-      - ${CI_PROJECT_DIR}/master_hash
 
 RunBenchmarks:
   extends: .LoadModules
   stage: runBenchmarks
   script:
-    - source ${CI_PROJECT_DIR}/master_hash
     - source ${CI_PROJECT_DIR}/branch_hash
     - export OMP_PROC_BIND=spread
     - export OMP_PLACES=threads
     - export JSRUN_OPTIONS="-n 1 -a 1 -c 42 -g 1 -r 1 -l CPU-CPU -d packed -b packed:42"
-    - export BENCHMARK_OPTIONS="--benchmark_repetitions=15
+    - export BENCHMARK_OPTIONS="--benchmark_repetitions=15 --benchmark_out_format=json
```

Comment:
I specifically decided not to put it there, figuring that benchmark options would only control the benchmarks we run, but not where and how we store the results.

Reply:
You seem to care more about that than me, so I'll change...